### PR TITLE
Credit Tweaks

### PR DESCRIPTION
**Pull request by FlattestGuitar**
:cl: Flattypatty
tweak: Increased end round credit speed, as well as Assistants being a separate group
/:cl:

Go commit by commit for review. I had the pleasure of learning a new thing about how BYOND treats indentation today. Turns out you can use as many tabs as you want for indentation as long as you keep it consistent within a proc or class definition. `proc/generate_credit_text()` was using two tabs instead of one and it compiled and ran just fine for whatever goddamn reason. I'm done.

I also ripped out the old script for generating new credits and the entire old system, since that's garbo.

Also, credits go a tiny bit faster now.

Also, credits now have a 1 tile margin on either side, as well as being more tightly packed together.

### CODE REVIEW

**Comment by AffectedArc07** [comment]
Images?

**Comment by SamHPurp** [comment]
@DaveTheHeadcrab feedback requested please

**Comment by SamHPurp** [comment]
>You'll notice that is why the credits hang near the top of the screen during highpop.

Ah hah, I was wondering if it was intentional!

**Comment by FlattestGuitar** [comment]
[here](https://www.dropbox.com/s/d0cpeomhalzieww/2018-04-03_15-10-35.webm?dl=0) is a sexy preview of the credits.

**Comment by DaveTheHeadcrab** [comment]
@FlattestGuitar 
That's pretty tight, make sure that the flukeops title card does not overlap the credits.

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -545,81 +545,94 @@
...
+	len_before_addition = round_credits.len
+	for(var/datum/mind/current in SSticker.mode.get_all_by_department(GLOB.supply_positions))
+		round_credits += "<center><h2>[current.name] as the [current.assigned_role]</h2>"
+	if(round_credits.len == len_before_addition)
+		round_credits += list("<center><h2>The station was freed from paperwork!</h2>", "<center><h2>No one worked in cargo today!</h2>")
+	round_credits += "<br>"
+
+	// CIVILIANS
+	var/list/human_garbage = list()
+	round_credits += "<center><h1>The Hardy Civilians:</h1>"
```
Comment (AffectedArc07) [inline code/game/gamemodes/game_mode.dm]: I would make this service crew

**Pull request merged**